src/kernels/FissionHeatSource.C

```diff
@@ -1,3 +1,6 @@
+/* This kernel will likely only be used with k-eigenvalue calculation mode with
+ * neutrons only, since the power is normalized in this case.  */
```

Comment:
Let's move this to the header file, and use Doxygen style commenting so that these will be automatically rendered on our web-site. Here's an example:
```c++
/**
 * This kernel implements the Laplacian operator:
 * $\nabla u \cdot \nabla \phi_i$
 */
class Diffusion : public Kernel
{
public:
  Diffusion(const InputParameters & parameters);

protected:
  virtual Real computeQpResidual() override;

  virtual Real computeQpJacobian() override;
};
```